### ISSUE TITLE
ItemInfo is missing opAssign

### ISSUE THREAD

**Issue by fnky** [baso88/SC_AngelScript#37]
ItemInfo is missing the opAssign method.

**Comment by fnky** [comment]
Code to fix this:

```cpp
static ItemInfo& ItemInfo_opAssign( const ItemInfo* pThis, const ItemInfo& other )
{
    *pThis = other;
    return *pThis;
}

//In RegisterSCScriptItemInfo:
pDocumentation->RegisterObjectMethod( "Assignment operator",
    pszObjectName, "ItemInfo& opAssign(const ItemInfo& in other)",
    asFUNCTION( ItemInfo_opAssign ), asCALL_CDECL_OBJFIRST );
```

In `trunk/dlls/Angelscript/ScriptTypes/Entity/ASPlayerItemEntity.cpp`

Untested, but should work fine. If not, adjust as needed.

**Comment by baso88** [comment]
The fix for this issue will be included in the next update (5.13). Closing.
